### ISSUE TITLE
The problem is... YOU!

### ISSUE THREAD

**Issue by Oracuse** [responsible-financial-innovation-act22/RFIA-bill#99]
Dear Congresswoman,

I have been in the crypto industry for 7 years as an educator, an advocate, and as a contributing member to the community, its advancement, and its proliferation. 7 years. I have colleagues whom I work with that have "enlightened" themselves less than 6 months ago. Their creative agility may be less than mine, simply because I have had more time to think over the different aspects and potential applications of essentially, decentralized infrastructure (meaning money and governance) - they are not behind me in terms of being knowledgeable about the technology and its potential uses. Real world, business-logic applied use cases. They can figure out what needs to be regulated, what will never be regulated, and what lies in between - even if those things are multi-dimensional (for instance, privacy, ownership, delegation, incentivization, etc). There is a saying in crypto, and that is that "everyone, is new to blockchain." We are all experimenting, innovating, and putting our skin in the game to bear risk. Because we understand, that the only way to push forward in this is to get involved. So we do it. Even at our own detriment. And the people who have believed that there are better ways to apply these technologies so we can collectively "revolutionize" the way we govern ourselves, also know that the current reality and status quo JUST DOESN'T CUT IT. Barring quantum computing to come save our civilization, we in this current system are running ourselves straight into the ground. It can be summed up in one word: Implosion.

My question to you, and all of you who are in positions of leadership and have been entrusted with the responsibility to steer the people, protect the interests of the greater good and well... lead is: Why can't you guys put in the time and effort to figure it out?! It is YOUR job to do so, isn't it? Don't get me wrong, I appreciate your call to "the plebs" for input and crowdsourcing your legal framework out to obtain as much feedback as possible - but isn't that like the CEO of a company asking their employees what they should do? The FACT that after all these years of free-market interactions in cryptocurrencies, the governing bodies STILL have not found a generally agreeable standard for regulation (or control, in another regard), proves one thing: you're just simply, unable to do it. There. I said it. 

Do you realize what that means? It means, you are passing on your responsibility to govern directly back to the people whom you govern. Yes, that is exactly what is going on - and its likely for the best. The centralization of your decision-making is a mirage. You've gone ahead and decentralized it, and are clinging to the notion that centralized governance in a capitalistic society still works; even when we have a president who refused to step down! The longer it takes for the governing bodies to realize this, the more unpleasantness there will be, and more people will get hurt from it. If you're no longer able to lead the people then get out of the way and help us to get to a better solution! There should be "A Committee for Decentralizing Governance" established with the task of making the transition in an orderly and methodical manner. Because if that is the way we're heading then let's get to it already. 

Sure, I could sit here and give you the answers you want all day long - but it isn't my responsibility to do so, nor have I been incentivized in any way shape or form. But I'll do it anyway, because perhaps I have something valuable to share. So unlike the leaders of the world today, who have value to share but DON'T, and now also have found ways to also not have to generate that value themselves... I can't be a hypocrite and hoard things when they are more valuable once given away. This is true of money, time, knowledge, and all other things that we hoard for our own personal use or gain. We are rivals for possession, and any society that can successfully flip this into anti-rivalry for possessions and properly incentivize the structure of that society - will make our current models of economics and governance completely obsolete. Like Neanderthals to Homo Sapiens. 

THERE ARE:
-Stores of value
-Payment currencies
-Security tokens
-Liquidity tokens (staking, yield-generating)
-Mining tokens
-Utility tokens
-Governance tokens
-Non-fungible tokens
-Non-transferable tokens 
-Permanence tokens (soulbound)
-Reputation tokens
-Asset-backed tokens
-Collateralized tokens
-Wrapped tokens
-Colored tokens
-Meme tokens
-Blockchain-agnostic tokens
-Reward tokens
-Fiat-backed Stablecoins
-Algorithmic Stablecoins
-Hybrid Stablecoins (or multi-asset)
-Bonded-curve tokens
-Privacy tokens (secret)
-Split tokens
-Fractional tokens (fungible non-fungible token)
-Carbon-credit tokens
-Synthetics
-Derivative tokens
-Insurance tokens
-Gaming tokens (game-fi)
-Storage tokens
-Data tokens
-Container tokens
-Earn tokens

and many more that I haven't thought of or are on the brink of innovation. The point is, the usage cases may require a different kind of token technology - and we don't know all the use cases yet. Not even close. And each type should and can be treated differently by the law. It will be larger than any other section of law, because by definition what tokens do is to mimic things that exist in a non-digital "world" (so that includes, tangible assets and intangible assets), as well as agreements (contract law) between two or more parties. That means that for every type of law we have, there is the potential to have a "digital" version of that same law. And more, since we are expanding the envelope of what is possible to do and achieve. 

By this logic, the attempts made by the SEC to "blanket" provision tokens and classify them as either security or not - is not only fundamentally insufficient; it is, rather extraordinarily oversimplified and lazy to even consider using any sort of law to cover all types of digital assets. Quite frankly, it has not been fair to the people who don't fall into these broad categories defined by regulations so far, to have to comply with rules that were written for something else (eg. how complicated and intricate is say, contract law?). And even more so, it has been unfair to those people who have lost money - not out of their own intent - because of the lack of such regulation in this extremely fast-growing market. What I'm talking about goes beyond scams, hacks, and fraudulence; I'm talking about the real ability of people to take advantage of others... by manipulating markets, using bot technology to front-run trades, and even insider trading, influencing the media, and the list goes on. Do you know about "MEV"? Learn what it is. See how in this decentralized, unregulated lawlessness, we sometimes just simply accept what's called "gaming the system." But even so, I think the general consensus is that we would RATHER be responsible for our own actions and consequences (ie. no regulation) than to have a body of lawmakers with a half-ass attempt at doing justice to what is really required of them. PLEASE DO YOUR JOBS, stop squabbling over things as if they were red & blue, as if you stood to gain or lose from the imposition of new regulations, and get the job done. I, we, understand - it is a lot to surmount. But that is exactly what is required of your position in society and responsibility to it. 

Yours sincerely,
Kawin R.
A self-appointed representative of the blockchain community
A leader and speaker for those without voices
An average dad trying to make a better life for his children

**Comment by S0ulf3re** [comment]
I don’t think I’ll be able to sleep at night anymore because of all the buzzwords in this post keeping me awake

**Comment by RocketPunches** [comment]
Wow, you came up with a list of 34 ways that cryptographic ledgers have been pointlessly tacked on to existing processes. Impressive.

**Comment by woot000** [comment]
I do not think "meme tokens" need to be or should ever be codified into United States law for any reason

**Comment by rickandm00rty** [comment]
It certainly might help to specify a sort of tier list for regulation. Which of these is most fundamental and important to start with? I agree with @woot000 - meme coin regulation should be not be codified into law, they are akin to art or speech

**Comment by steakknife** [comment]
TL;DR. WONTFIX. Sorry.

**Comment by kentkos** [comment]
Might as well sign your full name Kawin Rungsimuntakul. 

"It's not my responsibility and I was not incentivized".

Well Kawin, we in the United States don't require monetary compensation to participate in democracy.
